GVRf/Framework/src/org/gearvrf/asynchronous/GVRAsynchronousResourceLoader.java

```diff
@@ -159,6 +164,8 @@ public void run() {
                         });
                     } catch (Exception e) {
                         callback.failed(e, resource);
+                    } finally {
+                        resource.closeStream();
```

Comment:
I will check the other streams in the GVRAndroidResource and AsyncBitmapTexture scope